nextstrain/cli/argparse.py

```diff
@@ -157,43 +155,37 @@ def __call__(self, parser, namespace, value, option_string = None):
         setattr(namespace, self.dest, [*current, value])
 
 
-def runner_module(name: str) -> RunnerModule:
+def runner_module_argument(name: str) -> RunnerModule:
     """
-    Converts a string *name* into a :py:cls:`RunnerModule`.
+    Wraps :py:func:`runner_module` for friendlier error handling.
 
-    *name* is case-insensitive and underscores, hyphens, or spaces may be used
-    to separate words.  Internally, they're normalized to hyphens and the whole
-    string is lowercased.
-
-    Raises a :py:exc:`ArgumentTypeError` if *name* is unknown, which lets
+    Converts :py:exc:`ValueError` to :py:exc:`ArgumentTypeError`, which lets
     :py:mod:`argparse` emit a nice error message when this function is used as
     an argument ``type``.
 
-    >>> runner_module("docker") # doctest: +ELLIPSIS
+    >>> runner_module_argument("docker") # doctest: +ELLIPSIS
     <module 'cli.runner.docker' from '...'>
 
-    >>> runner_module("AWS-Batch") # doctest: +ELLIPSIS
+    >>> runner_module_argument("AWS-Batch") # doctest: +ELLIPSIS
     <module 'cli.runner.aws_batch' from '...'>
 
-    >>> runner_module("AWS Batch") # doctest: +ELLIPSIS
+    >>> runner_module_argument("AWS Batch") # doctest: +ELLIPSIS
     <module 'cli.runner.aws_batch' from '...'>
 
-    >>> runner_module("invalid")
+    >>> runner_module_argument("native") # doctest: +ELLIPSIS
+    <module 'cli.runner.ambient' from '...'>
+
+    >>> runner_module_argument("invalid")
     Traceback (most recent call last):
         ...
-    argparse.ArgumentTypeError: invalid runtime name: 'invalid'
+    argparse.ArgumentTypeError: invalid runtime name: 'invalid'; valid names are: 'docker', 'conda', 'ambient', 'aws-batch'
```

Comment:
17841d4 includes `ambient` in parts of this file when it should be `native` first then renamed in c7ee3cc.

Reply:
Whoops! Good catch. There was another anachronism in 8f1aa16 too. I was moving too fast at the end of the day! [Fixed with repush](https://tsibley.net/paste/2022-10-11IAmVQedu).